openrest4j-api/src/main/java/com/openrest/v1_1/Restaurant.java

```diff
@@ -43,6 +44,8 @@ public Restaurant(String id, String alias, String affiliateId, Map<String, Strin
         this.reservations = reservations;
         this.features = features;
         this.currentLocationId = currentLocationId;
+        this.defaultLocationIdAtTimeOfMigration = defaultLocationIdAtTimeOfMigration;
+        this.locationName = locationName;

```

Comment:
why is that required?

Reply:
If we'll need to return the title(business name)  and the locationName at the object.

Reply:
But why shouldn't it be added on the fly?
Why should we store it (what is the difference from what we said about currentLocationId)?

Reply:
right, i'll change it.

Reply:
Actually I think I get it...